java/dagger/android/internal/ProguardRulesReceiver.java

```diff
@@ -0,0 +1,17 @@
+package dagger.android.internal;
+
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+
+import static java.lang.annotation.ElementType.TYPE;
+import static java.lang.annotation.RetentionPolicy.SOURCE;
+
+/**
+ * Our proguard rules generator needs one annotation to hook into for it to run, so we use this
```

Comment:
Maybe we can make ProguardProcessor requests all annotations (returning `"*"` on `getSupportedAnnotations()`), that way we don't need to make a dummy annotation, since either way we always want to apply this processor.

Reply:
Good idea! I always forget that's possible 21b6d6e